cvcl/templates/course_detail.html

```diff
@@ -86,7 +86,7 @@
                 {% for student in course.students.all %}
                     <tr>
                         <td>{{ student }}
-                            <form action="{% url 'delete_user' user_pk=student.pk pk=course.pk %}" method="post">
+                            <form action="{% url 'delete_student' user_pk=student.pk pk=course.pk %}" method="post">
```

Comment:
This name is not entirely correct because you're not completely deleting a student.